src/internal_events/aws_ecs_metrics.rs

```diff
@@ -6,20 +6,50 @@ use metrics::{counter, histogram};
 use vector_core::internal_event::InternalEvent;
```

Comment:
Remember to drop the `skip check-events` comment at the top of the file when doing these updates.